presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/query/PinotQueryGeneratorContext.java

```diff
@@ -273,10 +302,15 @@ private boolean hasOrderBy()
         return variablesInAggregation;
     }
 
+    public PinotQueryGenerator.GeneratedPinotQuery toQuery(PinotConfig pinotConfig, ConnectorSession session)
+    {
+        return useSqlSyntax ? toSqlQuery(pinotConfig, session) : toPqlQuery(pinotConfig, session);
+    }
+
     /**
      * Convert the current context to a PQL
      */
-    public PinotQueryGenerator.GeneratedPql toQuery(PinotConfig pinotConfig, ConnectorSession session)
+    public PinotQueryGenerator.GeneratedPinotQuery toPqlQuery(PinotConfig pinotConfig, ConnectorSession session)
```

Comment:
I didn't truly understand why the PQL query generation cannot use `outputs` and still uses `selections`.

Reply:
just my lazy and afraid it may break existing user :) 
ok, I will update this logic.

Reply:
We have pretty good tests for PQL side -- so you shouldn't be able to break it :-)

Reply:
updated :)